local_db/models.py

```diff
@@ -12,7 +12,9 @@ def local_request_id():
     return str(ulid())
 
 
-class EnumField(models.TextField):
+# Without the "ignore" this raises: Missing type parameters for generic type "TextField"
+# https://github.com/typeddjango/django-stubs/issues/285
+class EnumField(models.TextField):  # type: ignore
```

Comment:
Should we be documenting the `# type: ignore` comments?  This comment's quite useful but doesn't tell us anything that we wouldn't know by removing it and seeing how mypy complains.

We should be consistent.

Reply:
Yes, we should in general. Although I've got an improved workaround for this specific case.